tests/tests_llist_search.c

```diff
@@ -0,0 +1,45 @@
+/*
+** EPITECH PROJECT, 2022
+** Teams
+** File description:
+** tests_llist_search
+*/
+
+#include "llist.h"
+#include <criterion/criterion.h>
+#include <string.h>
+
+static bool compare(const void *elem, const void *data)
```

Comment:
could have returned strcmp directly to get the difference

Reply:
This is a test to make sure it works properly.